src/Methods/Runes/GetBalancesIndex.ts

```diff
@@ -0,0 +1,55 @@
+import { method, NotFoundError } from "@valkyr/api";
+import Schema, { string } from "computed-types";
+
+import { runes } from "~Database/Runes";
+
+type RuneBalance = {
+  balance: string;
+  rune_metadata: {
+    divisibility: number;
+    symbol: string;
+    name: string; // with spacers
+  };
+};
+
+type RuneBalanceMap = {
+  [runeTicker: string]: RuneBalance;
+};
+
+export default method({
+  params: Schema({
+    address: string,
+  }),
+  handler: async ({ address }): Promise<RuneBalanceMap> => {
+    const balances = await runes.addressBalances(address);
+    if (balances.length === 0) {
+      throw new NotFoundError("Address has no balances");
+    }
+
+    const responseBalances: RuneBalanceMap = {};
+    for (const balance of balances) {
+      balance.amount = BigInt(balance.amount);
+      const rune = await runes.findRune(balance.runeTicker);
```

Comment:
Not ideal to have a database call within a loop, especially if an address can hold many runes.